lib/cog/startup.ex

```diff
@@ -0,0 +1,83 @@
+defmodule Cog.Startup do
```

Comment:
I was thinking that it would make organization a bit better. We are starting to get a good bit of logic in there so I thought it was time to break it up. I also thought that it might make it easier to refactor this bit in the future if we so desired such a thing. That said, I wouldn't be heartbroken to move it back if you think that's better. It's basically just copied and pasted from the top level module.